### ISSUE TITLE
Feature Request：Bulk Installation of Jar Files

### ISSUE THREAD

**Issue by linpxin** [nikita36078/J2ME-Loader#482]
As a amateur of jar games, I often install and play jar games. When there are many jar files to be installed, I click "add" button and pick up a jar file repeatedly which waste time, so I hope you can add a function -- bulk installation -- for J2ME Loader. Thanks.

**Comment by amarcioz** [comment]
I do not see the need for this function.

**Comment by imkrut** [comment]
> I do not see the need for this function.

I disagree. On some games it takes longer than usual to install, would be cool to be able to batch install games while doing something else in the background. Specially if you consider that in some cases there are multiple versions of the same game and it takes a while to find "the right one".

**Comment by PatrikPepega** [comment]
Same for me. I got around 5000 java games and i really want this feature. After 3 days i was only able to install 500 of them. 500!! It's such a waste of time. Please devs can you add this feature?

**Comment by CSan-Yume** [comment]
For now you can download this bulk Installer apk
https://github.com/MrsEWE44/J2ME-Loader/releases/download/V1.8.2/J2ME_Loader-1.8.2-open-release.apk
